docs/quick_start_guide.md

```diff
@@ -0,0 +1,63 @@
+# Quick Start Guide
+
+Whether you want to add Kotlin DL into your project, or experiment with it in a Jupyter Notebook, first you'll need to 
+set up appropriate dependencies.  
+
+## Working with Kotlin DL in an IDE
+1. Open a Kotlin project where you want to use Kotlin DL, or create a new Kotlin project in IntelliJ IDEA as described 
+in [Kotlin documentation](https://kotlinlang.org/docs/tutorials/jvm-get-started.html).
+2. Add the Kotlin DL dependency to your project's build file. 
+    * If you're using Gradle as build system, add the following to the `build.gradle.kts` file:
```

Comment:
Could you please add the instruction for pure Gradle (not kts) too

Reply:
Perhaps a stupid question, but is it going to be different?